build-system/tasks/helpers.js

```diff
@@ -507,10 +510,11 @@ async function compileTs(srcDir, srcFilename, destDir, options) {
  */
 async function compileJs(srcDir, srcFilename, destDir, options) {
   options = options || {};
+  // TODO
```

Comment:
?

Reply:
I was originally going to target minification, but it doesn't seem necessary now. Most consumer of npm packages run their own bundle+minfiy step when creating their binaries. Removed this TODO.